.env.example

```
@@ -24,6 +24,15 @@ WATCH_NAMESPACE="hades"
 DELETE_ON_COMPLETE="true"
 MAX_PARALLELISM="100"
 
+### HadesArtemisAdapter Options ###
+ARTEMIS_ADAPTER_PORT=8082 # Port for the Artemis Adapter to listen on (default: 8082)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)